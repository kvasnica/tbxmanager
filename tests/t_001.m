function t_001
% "tbxmanager" should:
% * print the help
% * set the path to pwd

onCleanup(@(x) tbx_setupTest('done')); tbx_setupTest('start');

tbxmanager

end
