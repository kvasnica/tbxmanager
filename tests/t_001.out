  Toolbox manager
 
  Supported commands:
    tbxmanager install package1 package2 ...
    tbxmanager show enabled
    tbxmanager show installed
    tbxmanager show available
    tbxmanager show sources
    tbxmanager update
    tbxmanager update package1 package2 ...
    tbxmanager restorepath
    tbxmanager enable package1 package2 ...
    tbxmanager disable package1 package2 ...
    tbxmanager uninstall package1 package2 ...
    tbxmanager source add URL
    tbxmanager source remove URL
    tbxmanager selfupdate
 
  For help, contact michal.kvasnica@stuba.sk
